### PR TITLE
S 05 all css

### PR DESCRIPTION
**Pull request by JulienBenoit7**

### CODE REVIEW

File: frontend/src/components/CategoryBar.css [hunk 1]
```diff
@@ -7,6 +7,11 @@
   padding: 1rem;
   justify-content: space-between;
   z-index: 50;
+  flex-direction: column;
+  top: 0;
```
Comment (rocambille) [inline frontend/src/components/CategoryBar.css]: Don't Repeat Yourself (and Don't Contradict Yourself ;) )

File: frontend/src/components/CategoryBar.css [hunk 2]
```diff
@@ -41,13 +46,18 @@
...
-/* Desktop styles */
-@media (min-width: 768px) {
+@media (max-width: 768px) {
   .category-bar {
-    flex-direction: column;
-    top: 0;
-    left: 0;
-    padding: 3em;
-    gap: 9rem;
+    display: flex;
```
Comment (rocambille) [inline frontend/src/components/CategoryBar.css:46]: Don't Repeat Yourself : what was said by default does'nt have to be said again in a media query

File: frontend/src/components/CategoryBar.css [hunk 3]
```diff
@@ -41,13 +46,18 @@
   color: white;
 }
 
-/* Desktop styles */
-@media (min-width: 768px) {
+@media (max-width: 768px) {
```
Comment (rocambille) [inline frontend/src/components/CategoryBar.css]: prefer mobile first : this should be min-width, not max-width

File: frontend/src/components/CategoryBar.css [hunk 4]
```diff
@@ -41,13 +46,18 @@
...
   .category-bar {
-    flex-direction: column;
-    top: 0;
-    left: 0;
-    padding: 3em;
-    gap: 9rem;
+    display: flex;
+    flex-direction: row;
+    position: fixed;
```
Comment (rocambille) [inline frontend/src/components/CategoryBar.css]: what's wrong with sticky position ?

File: frontend/src/components/CategoryBar.css [hunk 5]
```diff
@@ -41,13 +46,18 @@
...
+    bottom: 0;
+    padding: 1rem;
+    justify-content: space-between;
+    background-color: black;
+    width: 93%;
```
Comment (rocambille) [inline frontend/src/components/CategoryBar.css]: no fixed with

**Review by rocambille — CHANGES_REQUESTED** [review]

File: frontend/src/components/CategoryBar.jsx [hunk 6]
```diff
@@ -10,20 +10,20 @@ function CategoryBar() {
...
-            className="category-button"
-            to={`/category/${sport.name}`}
-          >
-            <span className="material-icons">{sport.icon}</span>
-            {sport.name}
-          </Link>
-        ))}
-      </div>
-    </nav>
+    // <nav>
```
Comment (Justine-Merlin) [inline frontend/src/components/CategoryBar.jsx]: Remove dead code

**Review by Justine-Merlin — CHANGES_REQUESTED** [review]

**Review by Justine-Merlin — APPROVED** [review]

**Pull request merged**